test_data/makedata_suites/102_collection_utf8.js

```diff
@@ -0,0 +1,301 @@
+    },
+    makeDataDB: function (options, isCluster, isEnterprise, database, dbCount) {
+      db._useDatabase('_system');
+      baseName = database;
+      if (baseName === "_system") {
+        baseName = "system";
+      }
+      baseName = `M${baseName}_${dbCount}_${extendedNames[3]}`;
+      print(`102: creating ${baseName}`);
+      db._createDatabase(baseName);
+    },
+    makeData: function (options, isCluster, isEnterprise, dbCount, loopCount) {
+      // All items created must contain dbCount and loopCount
+      // Create a few collections:
+      print(`102: using ${baseName}`);
+      db._useDatabase(baseName);
+      let c = createCollectionSafe(`c_${loopCount}${extendedNames[0]}`, 3, 2);
+      progress('createCollection1');
+      let chash = createCollectionSafe(`chash_${loopCount}${extendedNames[1]}`, 3, 2);
+      progress('createCollection2');
+      let cskip = createCollectionSafe(`cskip_${loopCount}${extendedNames[2]}`, 1, 1);
+      progress('createCollection3');
+      let cfull = createCollectionSafe(`cfull_${loopCount}${extendedNames[3]}`, 3, 1);
+      progress('createCollection4');
+      let cgeo = createCollectionSafe(`cgeo_${loopCount}${extendedNames[4]}`, 3, 2);
+      progress('createCollectionGeo5');
+      let cunique = createCollectionSafe(`cunique_${loopCount}${extendedNames[5]}`, 1, 1);
+      progress('createCollection6');
+      let cmulti = createCollectionSafe(`cmulti_${loopCount}${extendedNames[6]}`, 3, 2);
+      progress('createCollection7');
+      let cempty = createCollectionSafe(`cempty_${loopCount}${extendedNames[7]}`, 3, 1);
+
+      // Create some indexes:
+      progress('createCollection8');
+      createIndexSafe({col: chash, type: "hash", fields: ["a"], unique: false, name: extendedNames[1]});
+      progress('createIndexHash1');
+      createIndexSafe({col: cskip, type: "skiplist", fields: ["a"], unique: false, name: extendedNames[2]});
+      progress('createIndexSkiplist2');
+      createIndexSafe({col: cfull, type: "fulltext", fields: ["text"], minLength: 4, name: extendedNames[3]});
+      progress('createIndexFulltext3');
+      createIndexSafe({col: cgeo, type: "geo", fields: ["position"], geoJson: true, name: extendedNames[4]});
+      progress('createIndexGeo4');
+      createIndexSafe({col: cunique, type: "hash", fields: ["a"], unique: true, name: extendedNames[5]});
+      progress('createIndex5');
+      createIndexSafe({col: cmulti, type: "hash", fields: ["a"], unique: false, name: extendedNames[6]});
+      progress('createIndex6');
+      createIndexSafe({col: cmulti, type: "skiplist", fields: ["b", "c"], name: extendedNames[7]});
+      progress('createIndex7');
+      createIndexSafe({col: cmulti, type: "geo", fields: ["position"], geoJson: true, name: extendedNames[8]});
+      progress('createIndexGeo8');
+      createIndexSafe({col: cmulti, type: "fulltext", fields: ["text"], minLength: 6, name: extendedNames[0]});
+      progress('createIndexFulltext9');
+
```

Comment:
Please add inverted index here as well

Reply:
created subsequent QA-456 task for this.

Reply:
@dothebart I think you can copy definitions from 101 and 402 test suites. It doesn't look like a huge work to add missing stuff